src/mappings/ibc.ts

```diff
@@ -23,5 +23,8 @@ export async function handleMsgRecvPacket(msg:MsgRecvPacketMessage) {
 
 }
 
-
+export async function handleMsgAcknowledgement(msg:MsgAcknowledgementMessage):Promise<void> {
+  const transaction = createTransaction('MsgAcknowledgementMessage',msg)
```

Comment:
Parameter this `createTransaction` function must be same with the original message name in this case its `MsgAcknowledgement`.

Reply:
what about others `createTransaction`  need same edit now ?